FogCreek/Glitch-Community
# Community Infra connections for staging

## Links
* [README](ci/README.md)
* [diagram](https://docs.google.com/drawings/d/1KmJhEIkqhS2VBxhiZNYJSiO0iUX8IrVQW4soiHcE_0k/edit)

## Changes:
* updated `.circleci/config.yml` to add deploy steps
* added scripts in `ci` to handle CircleCI-based and local-to-the-community-hosts deploy scripts

## How To Test:
* merge changes into staging
* watch CircleCI job execute (and, hopefully, succeed!)
* check staging.glitch.com for update

## Feedback I'm looking for:
* is this understandable?
* is it clear how development will change?
* what additional docs or guidance is needed?
* is it stable?
* what else do we need to address (aside from moving into prod)?

## Things left to do before deploying:
* i think this can be merged and we can start testing it - it shouldn't affect anyone not merging to `staging`, which is new, so....
